no-computer.md

```diff
@@ -57,13 +98,27 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+Language, which involves defining vocabulary and rules, takes 90 percent of the
+time. Abstract games are all about language, and facilitating a way for players
+to communicate to each other to come closer to a winning end state. Decisions 
+involve things like how many colored pieces each player should get, how many 
+positions should be on the board, and the different ways a player is allowed to
+swap pieces. The rest of the time is spent basically designing the aesthetic of
+the board and pieces.
 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
+It's a relatively small idea that can be implemented with very few materials -
+just some colored paper and pen. The project is ambitious in the sense that
+it requires a lot of thinking to make a functional game that isn't impossible or
+too easy to win, and it requires a lot of playtesting. But overall it's very fun
+to implement and watch people play it.
 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
-
+This is a good idea for a project in that it requires few outside skills to implement
+and can be physically modified easily. It does require a lot of playtesting, which can 
```

Comment:
What does it mean for a game to require a lot of playtesting? What does playtesting achieve?